### PR TITLE
Update URLs after graduation

### PR DESCRIPTION
**Pull request by jezdez**
### Description

Update some URLs following #319.

### CODE REVIEW

File: conda_lock/_vendor/conda/cli/install.py [hunk 1]
```diff
@@ -292,11 +292,11 @@ def install(args, parser, command='install'):
                 # Other solvers might implement their own internal retry logic without
                 # depending --freeze-install implicitly like conda classic does. Example
                 # retry loop in conda-libmamba-solver:
-                # https://github.com/conda-incubator/conda-libmamba-solver/blob/da5b1ba/conda_libmamba_solver/solver.py#L254-L299
+                # https://github.com/conda/conda-libmamba-solver/blob/da5b1ba/conda_libmamba_solver/solver.py#L254-L299
```
Comment (ocefpaf) [inline conda_lock/_vendor/conda/cli/install.py:295]: @jezdez I believe you use some global search and replace here, right? These two URLs won't be the same.

Comment (maresb) [reply conda_lock/_vendor/conda/cli/install.py]: Agreed, and fixed in cb682df

**Comment by maresb** [comment]
Probably closes #326

**Review by maresb — APPROVED** [review]
I think this is ready to merge.

**Pull request merged**